lib/jekyll/configuration.rb

```diff
@@ -226,9 +226,10 @@ def backwards_compatibilize
       # Provide backwards-compatibility
       check_auto(config)
       check_server(config)
+      check_plugins(config)
 
       renamed_key "server_port", "port", config
-      renamed_key "plugins", "plugins_dir", config
+      renamed_key "plugins", "gems", config, false
```

Comment:
I think the order of the arguments is wrong here. The first argument is the old config, in this case `"gems"`. The second is the new config name, in this case `"plugins"`. 😄